app/pages/sections/additional-services/select/routes.test.js

```diff
@@ -343,7 +343,7 @@ describe('additional-services select routes', () => {
     ));
 
     it('should return the additional-services select recipient page if authorised', async () => {
-      getSolutionHelper.getSolution = jest.fn()
+      getRecipientsHelper.getRecipients = jest.fn()
```

Comment:
I've recently discovered a cleaner way of mocking a file that has a single function.

in the import leave this as;
`import { getRecipients } from '../../../../helpers/api/ordapi/getRecipients';`

add a jest.mock under the imports
`jest.mock('../../../../helpers/api/ordapi/getRecipients');`

Then when we need to mock it we can simply do
`getRecipients.mockResolvedValue({});`

Reply:
saves us having to alias it as something.